src/foam/nanos/u2/navigation/SignUp.js

```diff
@@ -226,67 +225,31 @@ foam.CLASS({
         return ! errors_ && ! isLoading_;
       },
       isAvailable: function(showAction) { return showAction; },
-      code: function(x, updateUser) {
+      code: function(x) {
         this.isLoading_ = true;
-
+        let createdUser = this.User.create({
+          userName: this.userName,
+          email: this.email,
+          desiredPassword: this.desiredPassword,
+          signUpToken: this.token_,
+          language: this.defaultUserLanguage()
+        });
+        if ( this.pureLoginFunction ) createdUser.emailVerified = true;
```

Comment:
@Nauna does this work? This property should be filtered out on client user creation.